html/index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<!-- Meta -->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta name="description" content="Bugmark is the fastest way to build issue futures into your App.">
+
+	<!-- Twitter Cards -->
+	<meta name="twitter:card" content="summary">
+	<meta name="twitter:site" content="https://bugmark.net">
+	<meta name="twitter:title" content="Bugmark">
+	<meta name="twitter:description" content="Bugmark is the fastest way to build issue futures into your App.">
+
+	<!-- Facebook Open Graph -->
+	<meta property="og:locale" content="en_US">
+	<meta property="og:url" content="https://bugmark.net">
+	<meta property="og:site_name" content="Bugmark">
+	<meta property="og:title" content="Bugmark">
+	<meta property="og:description" content="Bugmark is the fastest way to build issue futures into your App.">
+	<meta property="og:type" content="website">
+
+	<!-- Title -->
+	<title>Bugmark</title>
+
+	<!-- Fonts -->
+	<link href="https://use.fontawesome.com/releases/v5.0.8/css/all.css" rel="stylesheet">
+
+	<!-- CSS -->
+	<link rel="stylesheet" href="assets/vendor/bootstrap/dist/css/bootstrap.min.css">
+</head>
+
+<body>
+	<!-- navigation -->
```

Comment:
We could show the navigation in the same line of the logo/brand since all nav links will have relevance for the visitors, also we save space and get more fluid.